### PR TITLE
Add sample for load testing Azure SQL DB

### PR DESCRIPTION
**Pull request by ninallam**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Comment by ninallam** [comment]
@Nagarjuna-Vipparthi @ntrogh please review this for the upcoming blogpost.

File: jmeter-azure-services/azure-sql-database/SQLDB Test.jmx [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+        </collectionProp>
+      </Arguments>
+      <hashTree/>
+      <JDBCDataSource guiclass="TestBeanGUI" testclass="JDBCDataSource" testname="JDBC Connection Configuration" enabled="true">
+        <boolProp name="autocommit">true</boolProp>
+        <stringProp name="checkQuery"></stringProp>
+        <stringProp name="connectionAge">5000</stringProp>
+        <stringProp name="connectionProperties"></stringProp>
+        <stringProp name="dataSource">test_pool</stringProp>
+        <stringProp name="dbUrl">jdbc:sqlserver://alt-demo-server.database.windows.net:1433;database=alt-demoblog-db</stringProp>
```
Comment (ntrogh) [inline jmeter-azure-services/azure-sql-database/SQLDB Test.jmx:41]: Could you make this into a parameter?

File: jmeter-azure-services/azure-sql-database/SQLDB Test.jmx [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+        <stringProp name="driver">com.microsoft.sqlserver.jdbc.SQLServerDriver</stringProp>
+        <stringProp name="initQuery"></stringProp>
+        <boolProp name="keepAlive">true</boolProp>
+        <stringProp name="password">${udv_password}</stringProp>
+        <stringProp name="poolMax">0</stringProp>
+        <boolProp name="preinit">false</boolProp>
+        <stringProp name="timeout">10000</stringProp>
+        <stringProp name="transactionIsolation">DEFAULT</stringProp>
+        <stringProp name="trimInterval">60000</stringProp>
+        <stringProp name="username">azureuser</stringProp>
```
Comment (ntrogh) [inline jmeter-azure-services/azure-sql-database/SQLDB Test.jmx:51]: Can this also be a parameter? This will make the script more generic.

**Review by ntrogh — COMMENTED** [review]
Looks good to me. Just a suggestion to move the connection string and username into script parameters.

File: jmeter-azure-services/azure-sql-database/SQLDB Test.jmx
Comment (ntrogh) [inline jmeter-azure-services/azure-sql-database/SQLDB Test.jmx:1]: Could you change the file name to be all lowercase and not use spaces? It'll be more friendly on *nix.

File: jmeter-azure-services/azure-sql-database/Products.csv
Comment (ntrogh) [inline jmeter-azure-services/azure-sql-database/Products.csv:1]: Could you make the file name all lowercase to make it more *nix friendly?

**Pull request merged**